docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+1. I like to stay in VR and work in the virtual desktop to minimize headset fidgeting time. It's worth knowing there are several good VR desktop solutions available like [this one, ](https://www.vrdesktop.net/) [this one, ](https://store.steampowered.com/app/457550/Bigscreen_Beta/) [this one](https://store.steampowered.com/app/1068820/OVR_Toolkit/) and [this one](https://store.steampowered.com/app/1178460/Desktop_Portal/) that play well with SteamVR, some of which look and feel much better than the built-in one. 
+
+2. Coding is done in VS Code, with the `npm start` script running in VS Code's terminal (you can launch a new terminal instance from VS Code's Terminal tab):
+   
+![workflow-1](./media/0-0-workflow.jpg)
+
+Note that the script watches the files and recompiles on change, so you don't have to "refresh" anything aside from hitting save.
+
+2. When iterating, the Aardvark renderer does *not* have to be restarted to reflect gadget updates but you *do* have to instantiate a new gadget every time you want to see the latest compiled changes (and probably want to discard that gadget after testing).
+This is what my workflow typically looks like - IDE, test in SteamVR, back to IDE:
+
+![workflow.gif](./media/workflow.gif)
+
+3. For debugging, remember you can use Chrome's devtools on the Aardvark Server (http://localhost:8042/), as well as the Gadget Monitor (http://localhost:23842/gadgets/aardvark_monitor/index.html). You can also debug your gadget from VS Code, provided you've installed the Debugger for Chrome extension and answered "yes" to the `Do you want to debug with VS Code?` question when [instantiating your gadget](https://aardvarkxr.github.io/aardvark/getting_started/). 
+
+
+# Step 2: _Time of the Season_ - Adding a time/date panel
+
+Alright, now that we've got a dev environment and a nice workflow, we can jump into the code. 
+
+You can find and download the **necessary media assets** we'll be using [here](https://github.com/MichaelHazani/av-watch-gadget/tree/master/PROJECT_ASSETS). You'll notice that repo is the finished project, but I highly recommend starting from scratch and working your way through the tutorial). 
+
+Let's start by [instantiating a new gadget](https://aardvarkxr.github.io/aardvark/getting_started/). Give it any name you want, and select the defaults for everything (see the [previous tutorial](https://aardvarkxr.github.io/aardvark/getting_started/) for a recap if you'd like). 
+
+We'll be working primarily in `main.tsx` and `styles.css` located in the `src/` folder, which (after running `npm start` as described in the previous tutorial) get recompiled to the `dist/` folder after every save. This is an important fact to keep in mind; for one thing, you'll want to place whatever 3D models and various media files you'll be using in the `dist` folder since that'll be the root of your local references - where `main.tsx` will be solving paths from.
```

Comment:
I usually use file copy rules in webpackconfig.js for this that way the files can live in sec and get copied to dist whenever they change. There's a rule in the webpackconfig.js that the create script makes for the placeholder model.

Reply:
Ah, that's really helpful - I'll revise to reflect that workflow, thanks! Much cleaner than just working in `dist`.